src/utils/ohlc_fetcher.py

```diff
@@ -0,0 +1,149 @@
+import calendar
+import datetime as dt
+import json
+import time
+
+import pandas as pd
+import requests
+from sqlalchemy import create_engine, desc, select
+
+engine = create_engine(
+    "postgresql+psycopg2://Jayc:admin@127.0.0.1:5432/bigaluto_stockdb"
```

Comment:
encapsulate the connection string into environment environments with default value